From 864bc4835d25339527c71df6a4458073aab27b5f Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 17 Apr 2019 19:05:26 +0100
Subject: [PATCH 1/1] Set passthrough as default player input mode

* Supports all players without needing to change the input mode

Signed-off-by: Joey Riches <josephriches@gmail.com>
---
 src/app/data/models/settings/streaming/fragment.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/app/data/models/settings/streaming/fragment.js b/src/app/data/models/settings/streaming/fragment.js
index 98acf361..40af7022 100644
--- a/src/app/data/models/settings/streaming/fragment.js
+++ b/src/app/data/models/settings/streaming/fragment.js
@@ -32,7 +32,7 @@ export default Fragment.extend({
 
 	oauth: attr( "boolean", { defaultValue: true } ),
 	disable_ads: attr( "boolean", { defaultValue: false } ),
-	player_input: attr( "string", { defaultValue: ATTR_STREAMING_PLAYER_INPUT_STDIN } ),
+	player_input: attr( "string", { defaultValue: ATTR_STREAMING_PLAYER_INPUT_PASSTHROUGH } ),
 	player_no_close: attr( "boolean", { defaultValue: false } ),
 	hls_live_edge: attr( "number", { defaultValue: 3, min: 1, max: 10 } ),
 	hls_segment_threads: attr( "number", { defaultValue: 1, min: 1, max: 10 } ),
-- 
2.21.0

